compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1166,6 +1167,9 @@ static void addLowerToLLVMGPUPasses(OpPassManager &modulePassManager,
       // Barrier elimination before we reach unstructured control flow.
       .addPass(createGpuEliminateBarriers);
 
+  if (forROCDL)
+    funcPassManager.addPass(amdgpu::createAmdgpuMaskedloadToLoadPass);
```

Comment:
the iree coding standards require braces around single-statement `if`s/loops
```suggestion
  if (forROCDL) {
    funcPassManager.addPass(amdgpu::createAmdgpuMaskedloadToLoadPass);
  }
```